pageserver/src/http/routes.rs

```diff
@@ -3670,7 +3675,7 @@ pub fn make_router(
         })
         .get(
             "/v1/tenant/:tenant_shard_id/timeline/:timeline_id/getpage",
-            |r| testing_api_handler("getpage@lsn", r, getpage_at_lsn_handler),
+            |r| api_handler(r, getpage_at_lsn_handler),
```

Comment:
Promoting this from a testing-only API is a security risk, as the response includes the full page of user data.

I suggest we either make this API not return any user data (or perhaps just returns a hash?) when the `testing` feature is disabled, or add a separate "touch page" API that does the same lookup but doesn't actually return any data.

Reply:
added a separate touchpagelsn API